cmd/forwarder/fixer_test.go

```diff
@@ -105,6 +106,35 @@ func TestFixWithQueryParameters(t *testing.T) {
 	assert.Equal(int64(2), r.numLogs)
 }
 
+func TestFixWithCustomQueryParameters(t *testing.T) {
+	assert := assert.New(t)
+	var output = []byte("216 <13>1 2013-06-07T13:17:49.468822+00:00 host heroku web.7 - [origin ip=\"1.2.3.4\"][metadata@123 index=\"i\" source=\"s\" sourcetype=\"st\" fields=\"custom1=cq1,custom2=cq2,credential_deprecated=true,credential_name=cred\"] hi\n219 <13>1 2013-06-07T13:17:49.468822+00:00 host heroku web.7 - [origin ip=\"1.2.3.4\"][metadata@123 index=\"i\" source=\"s\" sourcetype=\"st\" fields=\"custom1=cq1,custom2=cq2,credential_deprecated=true,credential_name=cred\"] hello\n")
+
+	os.Setenv("LOG_ISS_FIELD_PARAMS", "custom1,custom2")
```

Comment:
As mentioned above, we should parameterize the `fix` method. Then we can avoid using `os.Setenv` here (instead, we'll just pass a parameter, which should already be an array of strings).